apps/modernization-ui/src/apps/search/investigation/investigationTermsResolver.ts

```diff
@@ -25,26 +25,42 @@ const processingStatus = fromSelectables('processingStatuses', 'PROCESSING STATU
 const notificationStatus = fromSelectables('notificationStatuses', 'NOTIFICATION STATUS');
 
 const investigationTermsResolver = (entry: InvestigationFilterEntry): Term[] => {
+    if (!entry) {
```

Comment:
same thing here I believe this wont be needed because the entry is mandatory as well